server/config.py

```diff
@@ -1,3 +1,4 @@
 import os
 
 TEAM_NAME = os.environ['TEAM_NAME']
+MOSHES_GOOGLE_API_KEY = 'AIzaSyAN3q6E8D80h38QQsiSWQ5keW6YSRMPqYc'
```

Comment:
I'm not sure yet how to set `env` variables within flask. I will do a better job in a later pr.

Reply:
assuming you have it in the env file. you can...

print(os.environ.get('MOSHES_GOOGLE_API_KEY'))